src/qml/QFieldSettings.qml

```diff
@@ -15,7 +15,7 @@ Page {
   property alias locatorKeepScale: registry.locatorKeepScale
   property alias numericalDigitizingInformation: registry.numericalDigitizingInformation
   property alias showBookmarks: registry.showBookmarks
-  property alias nativeCamera: registry.nativeCamera
+  property alias nativeCamera2: registry.nativeCamera2
```

Comment:
Amazing capability :+1 
All good to me.
just why `nativeCamera(2)` ?

Reply:
In order to reset the setting for everybody.